src/main/kotlin/no/nav/klage/clients/KlageDittnavAPIClient.kt

```diff
@@ -52,6 +54,17 @@ class KlageDittnavAPIClient(
             .block() ?: throw RuntimeException("Unable to register journalpost ID for anke in klage-dittnav-api.")
     }
 
+    fun setJournalpostIdInKlanke(klankeId: String, journalpostId: String) {
+        logger.debug("Registering journalpost ID for klanke in klage-dittnav-api. KlankeId ref: {}, journalpostId: {}", klankeId, journalpostId)
+        klageDittnavAPIWebClient.post()
+            .uri("klanker/$klankeId/journalpostid")
```

Comment:
Antar at endepunkt i klage-dittnav-api må opprettes her. Kan lage et for `getJournalpostForKlankeId` også?